src/sentry/sentry_metrics/client/snuba.py

```diff
@@ -8,7 +8,7 @@
 from sentry import quotas
 from sentry.sentry_metrics.client.base import GenericMetricsBackend
 from sentry.sentry_metrics.use_case_id_registry import UseCaseID
-from sentry.testutils.cases import BaseMetricsTestCase
+from sentry.testutils.cases import BaseMetricsTestCase  # NOQA:S007
```

Comment:
@getsentry/owners-snuba This client imports test suite code into code that could run in production. Perhaps we could find another solution to this?